frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -227,6 +235,17 @@ const filterByGroup = (items, filters) => {
   );
 };
 
+export const filterByGroupByDropdown = (items, groupBy) => {
```

Comment:
```suggestion
export const filterByGroup = (items, group) => {
```

Reply:
It's not really a filter though. It's a sort or grouping. I'd remove filter from this.

Reply:
Changed it to `groupItems`